pkg/virtual/framework/forwardingregistry/rest.go

```diff
@@ -31,20 +31,22 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
+type StorageAdapter func(schema.GroupResource, customresource.Store) customresource.Store
```

Comment:
am always falling over "adapter" as a term. What is an adapter? A wrapper has more meaning IMO as it is says something about structure.

Reply:
Happy to change it to whatever :)